packages/loaders/src/styled/StyledProgress.ts

```diff
@@ -30,15 +30,34 @@ interface IStyledProgressBackgroundProps {
 
 const PROGRESS_BACKGROUND_COMPONENT_ID = 'loaders.progress_background';
 
+const colorStyles = ({
+  theme,
+  color
+}: IStyledProgressBackgroundProps & ThemeProps<DefaultTheme>) => {
+  const backgroundColor = getColor({
+    theme,
+    hue: 'neutralHue',
```

Comment:
Should this inherit from a background variable? 🤔

Reply:
As discussed with Design, we're not tying this one to a variable, but rather build it from scratch.

Reply:
fwiw, I think @geotrev's point of tying this to `foreground.subtle` actually does make sense as the background of this loader actually functions as a subtle foreground color when `Progress` is applied to a page. Let's not make any adjustments now, but I want to reserve space to triple-check semantics with design before v9 wraps up.